learn/src/main/res/layout/item_course.xml

```diff
@@ -16,8 +16,8 @@
 
         <ImageView
             android:id="@+id/ivLogo"
-            android:layout_width="@dimen/pathway_logo_size"
-            android:layout_height="@dimen/pathway_logo_size"
+            android:layout_width="48dp"
```

Comment:
why we did not change pathway_logo_size variables value and instead hardcode it?
We could have also created a new variable if we are not supposed to change the existing variable's value

Reply:
@shainakhatun

Reply:
Resolved